pyccel/ast/c_concepts.py

```diff
@@ -57,6 +58,68 @@ def obj(self):
         """
         return self._obj
 
+
+class PointerCast(TypedAstNode):
+    """
+    A class which represents the casting of one pointer to another.
+
+    A class which represents the casting of one pointer to another in C code.
+    This is useful for storing addresses in a void pointer or for storing.
```

Comment:
Was this sentence truncated by accident?

Reply:
I think I just wrote the truncated version first for the "short summary" then wrote the longer version. The "in C code" isn't strictly necessary as it is implicit from the location of the class, however it is still useful information so I put it in the extended description only

Reply:
I was referring to the "or for storing" part. Storing what?

Reply:
Ah, oops. I don't remember what I was thinking of here. I have removed the snippet. 23885db6241bf170363effd1aa9e8dc7404bfe43